src/course-unit/add-component/messages.js

```diff
@@ -21,6 +21,40 @@ const messages = defineMessages({
     id: 'course-authoring.course-unit.modal.container.cancel.button.text',
     defaultMessage: 'Cancel',
   },
+  modalComponentSupportLabelFullySupported: {
+    id: 'course-authoring.course-unit.modal.component.support.label.fully-supported',
+    defaultMessage: 'Fully supported',
+  },
+  modalComponentSupportLabelProvisionallySupported: {
+    id: 'course-authoring.course-unit.modal.component.support.label.provisionally-support',
+    defaultMessage: 'Provisionally supported',
+  },
+  modalComponentSupportLabelNotSupported: {
+    id: 'course-authoring.course-unit.modal.component.support.label.not-supported',
+    defaultMessage: 'Not supported',
+  },
+  modalComponentSupportTooltipFullySupported: {
+    id: 'course-authoring.course-unit.modal.component.support.tooltip.fully-supported',
+    defaultMessage: 'Fully supported tools and features are available on edX, are '
+      + 'fully tested, have user interfaces where applicable, and are documented in the '
+      + 'official edX guides that are available on docs.edx.org.',
```

Comment:
Should this message be a hyperlink?

Reply:
These are simple tooltip messages, similar to how it was implemented in the full-screen editor of the problem.
<img width="1727" alt="Screenshot at Apr 03 08-57-20" src="https://github.com/openedx/frontend-app-course-authoring/assets/17108583/201bdd5b-859a-450c-a66a-b04966a8de11">